bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss

```diff
@@ -1,10 +1,19 @@
+// border
```

Comment:
This file should be under a `placeholder` (or maybe in the `mixin`) folder. You are no exporting variables.